polling_stations/apps/addressbase/management/commands/import_cleaned_addresses.py

```diff
@@ -6,17 +6,18 @@
 
 class Command(BaseCommand):
 
+    table_name = 'addressbase_address'
```

Comment:
Still reading this PR so you might address this later. Are you thinking about implementing something like the `auth.User` model whereby we can set the `Address` model in settings?

I'm thinking something like this in `settings.py`:

`address_model = 'polling_stations.models.AddressWithPollingStation`

If you think that's a good pattern then this line should be something like:

`get_address_model_somehow().objects.model._meta.db_table`

Reply:
Yeah - I like where you're going with this. I did try to think of a way to infer the table name from the model, but wasn't sure how to specify it. Having an app setting which defaults to `Address` but can be overridden with `AddressWithPollingStation` or whatever seems obvious now you say it - this is why code review is good :). That way we can set this once rather than having to pass it as a param in multiple places.

The issue I can see is that I think at the time the settings are parsed, Django hasn't loaded the ORM/models yet so I guess the setting will have to be something like `ADDRESS_MODEL = 'uk_geo_utils.models.Address'` as opposed to `ADDRESS_MODEL = Address()` and then I'll have to go from the string representation to a class (if you see what I mean). I'm not 100% sure what the implementation looks like at the moment but I'll look into it as I think that a better approach.

Reply:
It might be worth researching 'swappable' https://code.djangoproject.com/ticket/19103 https://github.com/wq/django-swappable-models, but IDK. Django does this for `auth.User` and `Profile` but I've seen other apps do this sort of thing too (I'll try to remember some examples)